src/test/datascience/kernelProcess.vscode.test.node.ts

```diff
@@ -1,252 +0,0 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
My suspicion provide right, everything here is mocked and managed to convert this into a unit test without any changes except for change to the file extension.